src/linker/utilities/paths.py

```diff
@@ -3,4 +3,6 @@
 # TODO: We'll need to update this to be more generic for external users and have a way of configuring this
 CONTAINER_DIR = "/mnt/team/simulation_science/priv/engineering/er_ecosystem/images"
 IMPLEMENTATION_METADATA = Path(__file__).parent.parent / "implementation_metadata.yaml"
-LINKER_TEMP = Path("/tmp/linker")
+# Bind linker temp dir to /tmp in the container. Slurm will delete /tmp after job completion,
```

Comment:
> Slurm will delete /tmp after job completion

I don't think this is true (in fact have been annoyed by the opposite behavior)

Reply:
Hmm, documentation i read implied that was the case, but I think you are right and it depends on how the cluster is configured. I'm guessing the compute node's /tmp dir is cleaned up on some regular interval.

Reply:
Yeah it's totally cluster-specific. At IHME it's only cleared every 10 days or something, so it is actually quite easy to fill it up accidentally.